nmostesting/suites/IS0402Test.py

```diff
@@ -30,6 +30,7 @@
 from ..IS04Utils import IS04Utils
 from ..TestHelper import is_ip_address, load_resolved_schema, WebsocketWorker
 from ..TestResult import Test
+from ..mocks.Auth import PRIMARY_AUTH
```

Comment:
As for IS0401, the PRIMARY_AUTH should be injected in the constructor for consistency

Reply:
Done